index.html

```diff
@@ -67,6 +67,10 @@
                 class="u-active-grey-5 u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-hover-grey-10 u-nav-link u-text-active-grey-90 u-text-grey-90 u-text-hover-grey-90"
                 href="index.html" style="padding: 10px 20px;">Home</a>
             </li>
+            <li class="u-nav-item"><a
+              class="u-active-grey-5 u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-hover-grey-10 u-nav-link u-text-active-grey-90 u-text-grey-90 u-text-hover-grey-90"
+              href="login.html" style="padding: 10px 20px;">Login</a>
```

Comment:
Create a separate login/Signup tappable text that is not in the top navbar .. but somewhere down, below "Go  to recipe" button.
Remove the button from the top navbar

Reply:
![image](https://user-images.githubusercontent.com/63389169/196369041-5ab993d5-8d15-45fd-980d-428d94a44412.png)


![image](https://user-images.githubusercontent.com/63389169/196369207-f7a3e029-0f94-48ba-8e5d-b8d0ae9afeea.png)

you want things like these.

Reply:
Yes Perfect , just add one little detail and we're good to go :)
![image](https://user-images.githubusercontent.com/78858516/196373091-af1acdc1-cfe2-4881-b815-3b6cee1ff3a1.png)

 - Add a "Login/Sign Up" text down below the "Go to Recipes" button linking the login page.

Reply:
Make it a small tappable text , like the one found in your login.html file
eg.)
![image](https://user-images.githubusercontent.com/78858516/196374929-54c906f9-7c84-4426-aa7f-0e25998a9814.png)

Reply:
The same text you want on home page that's in the picture above or any specific text 
![image](https://user-images.githubusercontent.com/63389169/196377960-5647b931-bac2-4652-8546-915445d358fd.png)
or just like this?

Reply:
![image](https://user-images.githubusercontent.com/63389169/196378316-16426214-8c32-4018-972a-16f4df275e5c.png)

Reply:
Perfect

Reply:
Pushed all the changes in my login_branch and I am resolving this conversation and thanks for helping me out sir